CMakeLists.txt

```diff
@@ -477,10 +477,15 @@ sdl_glob_sources(
   "${SDL3_SOURCE_DIR}/src/stdlib/*.c"
   "${SDL3_SOURCE_DIR}/src/thread/*.c"
   "${SDL3_SOURCE_DIR}/src/timer/*.c"
-  "${SDL3_SOURCE_DIR}/src/video/*.c"
   "${SDL3_SOURCE_DIR}/src/video/yuv2rgb/*.c"
 )
 
+# Exclude intrinsinc modules from general builds
+file(GLOB video_sources "${SDL3_SOURCE_DIR}/src/video/*.c")
+list(FILTER video_sources EXCLUDE REGEX "SDL_blit_A_avx2.c$")
+list(FILTER video_sources EXCLUDE REGEX "SDL_blit_A_sse4_1.c$")
```

Comment:
It looks like you propertly guarded both files with `#ifdef`s, so you can keep both files in the glob.

Reply:
Thank you, I cleaned this up